libs/trpc/.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+	"extends": ["../../.eslintrc.json"],
+	"ignorePatterns": ["!**/*"],
+	"overrides": [
+		{
+			"files": ["*.ts"],
+			"rules": {
+				"@angular-eslint/directive-selector": [
+					"error",
+					{
+						"type": "attribute",
+						"prefix": "spartan-ng",
+						"style": "camelCase"
+					}
+				],
+				"@angular-eslint/component-selector": [
+					"error",
+					{
+						"type": "element",
+						"prefix": "spartan-ng",
```

Comment:
Suggestion: We could use something like `spartan-trpc` or `ng-trpc` as the prefix.

Reply:
I don't think we will add components or directives here so we can keep that for now!